components/higher-order/with-filters/test/index.js

```diff
@@ -82,12 +111,15 @@ describe( 'withFilters', () => {
 		);
 		const EnhancedComponent = withFilters( hookName )( SpiedComponent );
 
-		wrapper = mount( <EnhancedComponent /> );
+		wrapper = TestUtils.renderIntoDocument( getTestComponent( EnhancedComponent ) );
```

Comment:
Can we use destructuring when importing TestUtils to avoid using the variable??

Reply:
TestUtils is the default export from `react-dom/test-utils`.  So I'd have to destructure on a subsequent statement.  If `renderIntoDocument` was available as a direct import I'd definitely do it but its not.  For now, I think it's a bit useful knowing at a glance that `renderIntoDocument` is part of TestUtils?

Reply:
It’s quite surprising but good yo know 😃